### PR TITLE
Allow reply to address to be set

### PR DESCRIPTION
**Pull request by Henk3000**
You are able to use the reply to option in combination with a org wide email address, therefore I moved it.

https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_email_outbound_base.htm#!

### CODE REVIEW

**Comment by alexed1** [comment]
Nice! @jdpond can you take a quick look?

File: flow_action_components/FindCommonAndUniqueRecords/FindCommonAndUniqueRecords.cls [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/**
```
Comment (alexed1) [inline flow_action_components/FindCommonAndUniqueRecords/FindCommonAndUniqueRecords.cls:1]: @adwhite75  i think this was added to the wrong pull request. can you remove?

Comment (adwhite75) [reply flow_action_components/FindCommonAndUniqueRecords/FindCommonAndUniqueRecords.cls]: Ugh, sorry, yep! All done.

**Comment by alexed1** [comment]
@jdpond I'd love to get your quick look on this before I merge it in...

**Comment by alexed1** [comment]
Merging this in.

**Review by alexed1 — APPROVED** [review]

**Comment by jdpond** [comment]
A little late, didn't see this one, but yes, the change is correct

**Pull request merged**